docs/pages/versions/unversioned/sdk/sqlite.mdx

```diff
@@ -358,6 +359,58 @@ export default function App() {
 }
 ```
 
+### Sharing a database between apps/extensions (iOS)
+
+To share a database with other apps/extensions in the same App Group you can use shared containers. Follow these steps:
```

Comment:
```suggestion
To share a database with other apps/extensions in the same App Group, you can use shared containers by following the steps below:
```

Update this to avoid using a fragmented second sentence so the readability is improved.